### ISSUE TITLE
[v8] Scroll to story when clicking on search item

### ISSUE THREAD

**Issue by tlow92** [storybookjs/react-native#644]
**Is your feature request related to a problem? Please describe.**
When clicking on a search item the story opens but you can't see it in the list because the list does not scroll to the appropriate story. This is annoying because oftentimes you just search for a base story, but actually want to use a sibling of it or switch between them

**Describe the solution you'd like**
Automatically scroll the sidebar so that the selected story is in view

**Are you able to assist bring the feature to reality?**
Yes, I might implement this when I find time.

**Comment by dannyhw** [comment]
I agree, I had a hard time getting this to work initially so I pushed it off for later 😅

**Comment by dannyhw** [comment]
this should be working now if you install the latest version
